### PR TITLE
Fix mistake in overrides

### PR DESCRIPTION
**Pull request by fahhem**
Sorry for that!

### CODE REVIEW

**Comment by ali5h** [comment]
what is fixed?

**Comment by fahhem** [comment]
It turns out this wasn't working at all, I'll add a test that overrides pytz with something different and ensure it's not actually pytz

**Comment by fahhem** [comment]
okay, it's ready!

**Comment by fahhem** [comment]
@ali5h Any chance you have time to review this? I'm in no rush, we're using this branch internally and it's working great, but I figured since the feature's broken and part of the last release, it would be good to get this in and make a new release

File: examples/tests/test_import.py [hunk 1]
```diff
@@ -17,3 +17,5 @@ def test_numpy():
 
 def test_pytz():
     import pytz
+
```
Comment (ali5h) [inline examples/tests/test_import.py:20]: would you add a comment on this test

File: examples/WORKSPACE [hunk 2]
```diff
@@ -58,6 +58,7 @@ py_library(
     imports = ["."],
 )
 """,
+    patches = ["@//:pytz.new_attr.patch"],
```
Comment (ali5h) [inline examples/WORKSPACE:61]: also on why we have this patch

**Review by fahhem — COMMENTED** [review]
Added the comments, thanks!

**Pull request merged**